src/ac_training_lab/A1-cam/device.py

```diff
@@ -0,0 +1,118 @@
+import paho.mqtt.client as mqtt
+import boto3
+import json
+from datetime import datetime
+import time
+
+from picamera2 import Picamera2
+from libcamera import controls, Transform
+
+from my_secrets import (
+    CAMERA_READ_ENDPOINT,
+    CAMERA_WRITE_ENDPOINT,
+    MQTT_PORT,
+    MQTT_HOST,
+    MQTT_PASSWORD,
+    MQTT_USERNAME,
+)
+
+import logging
+import sys
+
+logging.basicConfig(
```

Comment:
Could you comment on the reasoning behind the more involved logging setup? (file handler, stream handler, why default `basicConfig` doesn't suffice, etc.). I imagine something to do with the Pi using a headless OS and having a file in case of unexpected shut-down, while still being able to see the stdout when you're SSH'd into it and debugging/prototyping.